tests/integration/replay/replay_tests.cpp

```diff
@@ -72,11 +57,13 @@ class replayTest {
       "thread_safe_allocator", rm.getAllocator("HOST"));
     allocatorNames.push_back("thread_safe_allocator");
 
+#if 0
```

Comment:
Why is this `#if 0`?

Reply:
It has been there ever since we realized that replay cannot support FixedBlock allocations.  This will become problematic with the upcoming changes for the pool of fixed pool strategy.

Reply:
Got it 👍

Reply:
I can remove this entirely if you like.  Or, I can ifdef it some other way and add a comment speaking to the limitation (probably the best idea)

Reply:
Explaining the limitation would be a good idea.

Reply:
Done, will you resolve this?